README.md

```diff
@@ -278,9 +277,9 @@ So let's get to the work! :mans_shoe:
 
 > I have also slightly altered nightwatch.json and package.json, so be sure to check it.
 
-Our tools creates for us testing instance, file with variables, but how we can use it? Lets consider a case, where we want to test our Script Endpoint socket in dashboard. They require that user created script (component) before creating Script Endpoint. How we can solve this issue? We could just create one more test case in our test suite for Script Endpoint. But... it won't be good idea in long run. We could easily duplicate code that way, create unnecessary more steps that take longer to execute, and we would have to do cleanup after each test suite.
+Our tool creates us a testing instance, file with variables, but how can we use it? Lets consider a case, where we want to test our Script Endpoint socket in the Dashboard. The tests require that the user has a `script` (component) before he can make a Script Endpoint. How we can solve this issue? We could just create one more test case in our test suite for Script Endpoint. But... it won't be good idea in the long run. We could easily duplicate code that way, create unnecessary more steps that take longer to execute, and we would have to do cleanup after each test suite.
 
-That's why we have created our tool, first create file `testScriptEndpoint.js` in tests folder, then this is how I'd used it in test:
+That's why we have created our tool, first create file `testScriptEndpoint.js` in the tests folder. This is how the test code should look like:
```

Comment:
Thus section should be under a separate title:
### Using the data in tests
or smth like that